components/Button.tsx

```diff
@@ -1,16 +1,17 @@
 interface props {
   value: string
+  buttonStyles?: string
 }
```

Comment:
Interface name should be start with capital letter, change this to `ButtonProps`